src/xtgeo/well/_wellmarkers.py

```diff
@@ -21,8 +21,6 @@ def get_zonation_points(self, tops, incl_limit, top_prefix, zonelist, use_undef)
 
     Args, see calling routine
     """
-
-    zlist = []
```

Comment:
This change is under the "Improve tests" commit, but seems to be a removal of unused variable.